src/utils/tooltip.class.js

```diff
@@ -836,26 +829,21 @@ class ToolTip {
 
   /* istanbul ignore next */
   setOnTouchStartListener(on) {
-    // if this is a touch-enabled device we add extra
-    // empty mouseover listeners to the body's immediate children;
-    // only needed because of broken event delegation on iOS
+    // If this is a touch-enabled device we add extra
+    // empty mouseover listeners to the body's immediate children
+    // Only needed because of broken event delegation on iOS
     // https://www.quirksmode.org/blog/archives/2014/02/mouse_event_bub.html
     if ('ontouchstart' in document.documentElement) {
       arrayFrom(document.body.children).forEach(el => {
         if (on) {
-          eventOn(el, 'mouseover', this._noop)
+          eventOn(el, 'mouseover', noop)
```

Comment:
Actually we can't use this here.... as we do an off below...  and all instances of tooltip and popwer would reference the same function...

So the `off` would turn it off for all popovers and tooltips.

To get around this we could do this:

```js
eventOn(el, 'mouseover', noop.bind(this))
//...
eventOff(el, 'mouseover', noop.bind(this))
```

To make sure only the instances "version" of the event is used.

Reply:
Although I'm not even sure if this will create a "new" reference to the function or not.

Reply:
As, when removing the event, we need to pass the same function reference to the `Off` as we did the `On`

Reply:
Which is why it was originally set to an instance no op mentod (`this.noop`)